wgpu-hal/src/vulkan/mod.rs

```diff
@@ -602,13 +602,6 @@ struct RenderPassKey {
     multiview: Option<NonZeroU32>,
 }
 
-#[derive(Clone, Eq, Hash, PartialEq)]
-struct FramebufferKey {
-    raw_pass: vk::RenderPass,
-    attachments: ArrayVec<vk::ImageView, { MAX_TOTAL_ATTACHMENTS }>,
-    extent: wgt::Extent3d,
-}
-
```

Comment:
question(non-blocking): Was there a reason you moved `FramebufferKey`?

Reply:
I moved it closer to where it will be used (in the command encoder).